### ISSUE TITLE
«JCB, Discover, and Diners Club cards can only be charged in USD»

### ISSUE THREAD

**Issue by dmitry-fedyuk** [mage2pro/stripe#28]
https://stripe.com/docs/currencies#charge-currencies

**Comment by dmitry-fedyuk** [comment]
I tried to charge [a test Discover bank card](https://stripe.com/docs/testing#cards) (`6011111111111117`) in `GBP`, and it really leads to an error:
```
{
    "error": {
        "charge": "ch_1BBpYGFzKb8aMux1p59GEgYO",
        "code": "card_declined",
        "decline_code": "card_not_supported",
        "message": "Your card is not supported. Please use a Visa, MasterCard, or American Express card",
        "param": "number",
        "type": "card_error"
    }
}
```

**Comment by dmitry-fedyuk** [comment]
Fixed in [2.0.0](https://github.com/mage2pro/stripe/releases/tag/2.0.0).
